src/mods/common/analyzer/append/bmesh.types.mod.rst

```diff
@@ -17,7 +17,7 @@
 
    .. method:: __iter__()
 
-      :rtype: BMIter[GenericType1]
+      :rtype: :class:`BMIter`\ [GenericType1]
```

Comment:
Why is `\` required?

Reply:
From the discussion on the Discord.

> In the [sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup):
> > [inline markup] must be separated from surrounding text by non-word characters
> I would have thought a [ would count as a non-word character, but I guess not. An escaped space \  before the bracket solves all.

But, I'm not sure that only one space is not enough.

Reply:
The example in the linked docs section is that `thisis\ *one*\ word` becomes thisis*one*word, i.e. the spaces exist for the purpose of separating the inline markup from word characters, but don't appear in the end result. As there shouldn't be a space between the class and the subscription brackets, I have escaped the space.

Reply:
I got it. Thank you!